partiql-tests-data/fail/static-analysis/primitives/timestamp-constructor.ion

```diff
@@ -0,0 +1,655 @@
+  assert: {
+    result: StaticAnalysisFail
+  },
+}
+
+{
+  name: "TIMESTAMP - WITH OUT TIMEZONE - RFC 3339 Fomat - invalid DAY field - out of range for April",
+  statement: "TIMESTAMP '2021-04-31T00:00:00'",
+  assert: {
+    result: StaticAnalysisFail
+  },
+}
+
+{
+  name: "TIMESTAMP - WITH OUT TIMEZONE - RFC 3339 Fomat - invalid DAY field - out of range for May",
+  statement: "TIMESTAMP '2021-05-32T00:00:00'",
+  assert: {
+    result: StaticAnalysisFail
+  },
+}
+
+{
+  name: "TIMESTAMP - WITH OUT TIMEZONE - RFC 3339 Fomat - invalid DAY field - out of range for June",
+  statement: "TIMESTAMP '2021-06-31T00:00:00'",
+  assert: {
+    result: StaticAnalysisFail
+  },
+}
+
+{
+  name: "TIMESTAMP - WITH OUT TIMEZONE - RFC 3339 Fomat - invalid DAY field - out of range for July",
+  statement: "TIMESTAMP '2021-07-32T00:00:00'",
+  assert: {
+    result: StaticAnalysisFail
+  },
+}
+
+{
+  name: "TIMESTAMP - WITH OUT TIMEZONE - RFC 3339 Fomat - invalid DAY field - out of range for August",
+  statement: "TIMESTAMP '2021-08-32T00:00:00'",
+  assert: {
+    result: StaticAnalysisFail
+  },
+}
+
+{
+  name: "TIMESTAMP - WITH OUT TIMEZONE - RFC 3339 Fomat - invalid DAY field - out of range for September",
+  statement: "TIMESTAMP '2021-09-31T00:00:00'",
+  assert: {
+    result: StaticAnalysisFail
+  },
+}
+
+{
+  name: "TIMESTAMP - WITH OUT TIMEZONE - RFC 3339 Fomat - invalid DAY field - out of range for October",
+  statement: "TIMESTAMP '2021-10-32T00:00:00'",
+  assert: {
+    result: StaticAnalysisFail
+  },
+}
+
+{
+  name: "TIMESTAMP - WITH OUT TIMEZONE - RFC 3339 Fomat - invalid DAY field - out of range for November",
+  statement: "TIMESTAMP '2021-11-31T00:00:00'",
+  assert: {
+    result: StaticAnalysisFail
+  },
+}
+
+{
+  name: "TIMESTAMP - WITH OUT TIMEZONE - RFC 3339 Fomat - invalid DAY field - out of range for December",
+  statement: "TIMESTAMP '2021-12-32T00:00:00'",
+  assert: {
+    result: StaticAnalysisFail
+  },
+}
+
+{
+  name: "TIMESTAMP - WITH OUT TIMEZONE - RFC 3339 Fomat - invalid DAY field - day of zero",
+  statement: "TIMESTAMP '2021-01-00T00:00:00'",
+  assert: {
+    result: StaticAnalysisFail
+  },
+}
+
+{
+  name: "TIMESTAMP - WITH OUT TIMEZONE - RFC 3339 Fomat - invalid DAY field - out of range",
+  statement: "TIMESTAMP '2021-03-12345T00:00:00'",
+  assert: {
+    result: StaticAnalysisFail
+  },
+}
+
+{
+  name: "TIMESTAMP - WITH OUT TIMEZONE - RFC 3339 Fomat - invalid Day field - minus before day",
+  statement: "TIMESTAMP '2021-03--10T00:00:00'",
+  assert: {
+    result: StaticAnalysisFail
+  },
+}
+
+{
+  name: "TIMESTAMP - WITH OUT TIMEZONE - RFC 3339 Fomat - invalid Day field - plus before day",
+  statement: "TIMESTAMP '2021-03-+10T00:00:00'",
+  assert: {
+    result: StaticAnalysisFail
+  },
+}
+
+{
+  name: "TIMESTAMP - WITH OUT TIMEZONE - RFC 3339 Fomat - invalid literal Format - random string",
+  statement: "TIMESTAMP 'timestamp'",
+  assert: {
+    result: StaticAnalysisFail
+  },
+}
```

Comment:
Test is the same as "TIMESTAMP - WITH OUT TIMEZONE - SQL Fomat - invalid literal Format - random string" since the string literal is the same. Can be deleted.